lib/active_merchant/billing/gateways/plexo.rb

```diff
@@ -207,6 +207,7 @@ def add_payment_method(post, payment, options)
 
           add_card_holder(post[:paymentMethod][:Card], payment, options)
         end
+        post[:paymentMethod][:Card][:Cryptogram] = payment.payment_cryptogram if payment&.is_a?(NetworkTokenizationCreditCard)
```

Comment:
❓ Question

Did you consider to add cryptogram to the scrub method, I can see some other gateways doing it.